<html>
<!--
-->

<head>
<title>Nomenclature for the description of sequence variations; History</title>
<style type="text/css">

<!--  

h1 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h2 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #ff9900;

}

h3 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h4 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h5 {
font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;
color : #ffffff;

}

p { 
font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;
color : #000000;
margin-left : 5%;

 }

.norm {

 font-weight : normal;

 }
 -->
</style>
</head>

<body text="#000000" bgcolor="#FFFFFF" link="#0000FF" vlink="#9999ff">

<table width="100%" border="0" cellspacing="0" cellpadding="10" bgcolor="#CCFF99">
  <tr>
    <td width="25%" valign="top" bgcolor="#FFFFFF"><a href="http://www.HGVS.org"><img src="small.gif" border="0" width="155" height="94"></a></td>
    <td width="75%" valign="CENTER" bgcolor="#FFFFFF" bordercolor="#FFFFFF"><h2 align="center"><font color="#0099CC">History:
      <a href="http://ariel.ucs.unimelb.edu.au/~cotton/antonara.htm">Link to
      original file</a> </font> </h2>
    </td>
  </tr>
</table>

<!--	Author: 	Dr. Johan T. den Dunnen	Date: 	08-05-01	URL: 	http://www.dmd.nl/index_discussion.html-->

<p><i>Draft 3; 29Nov96&nbsp;</i></p>

<p><b>RECOMMENDATIONS FOR A NOMENCLATURE SYSTEM FOR HUMAN GENE MUTATIONS</b></p>

<p>Prepared by Stylianos E Antonarakis</p>

<p><b>Introduction</b></p>

<p>This document is created as a result of the meeting on "locus-specific&nbsp;mutation databases" held on March 24, 1996 in Heidelberg, Germany and
the "Mutation Database" meeting on October 29, 1996 in San Francisco,
California. As a chairperson of the Nomenclature committee, I had
accepted the task preparing and circulating a document with
recommendations for debate, further discussions and most importantly for
final approval / acceptance at the October 1996 meeting in San
Francisco. The first draft of this document (5 Aug 96) was distributed&nbsp;to a number of investigators, the majority of which are co-authors in
the Beaudet et al and Beutler et al papers (see below). This second
draft (5 Sep 96) contained modifications according to their suggestions,
opinions and criticisms. Furthermore, many colleagues had offered
suggestions, criticisms, and ideas through electronic communication.
During the October 29, 1996 meeting in San Francisco, there was a
sufficient discussion of these recommendations and it was agreed that
another, further modified draft of the document would be posted on the
"World Wide Web" for final debate for a period of 6 weeks. Then, this
set of initial recommendations would be published as guidelines for
mutation nomenclature.&nbsp;&nbsp;<br>
<br>
Two manuscripts were recently published in the journal "Human Mutation"
that contain mutation nomenclature recommendations: "Update on
nomenclature for human gene mutations" by Beaudet et al., Hum.Mut. 8:
197-202, 1996 and "Mutation nomenclature: nicknames, systematic names
and unique identifiers" by Beutler et al., Hum.Mut. 8: 203-206, 1996.
These documents present the views of the authors after discussions
during the October 19, 1994, Montreal and the October 24-25, 1995
Minneapolis meetings. Other published papers/letters on nomenclature
issues include the following: Beaudet &amp; Tsui, Hum.Mut. 2: 245, 1993; Beutler, Am.J.Hum.Genet. 53: 783, 1993; Antonarakis &amp;
McKusick, Hum.Mut.
4: 166, 1994.&nbsp;<br>
<br>
It is obvious that the most unambiguous nomenclature system is that
based on genomic DNA. Even in that case, however, length polymorphisms
create a problem in the numbering of nucleotides and therefore a
standard, reference sequence ought to be established, preferably by the
experts. Unfortunately, the entire genomic sequence is only known for a
minority of human genes. For the vast majority of genes, the known
sequence is that of cDNA. The existence of more than one transcription
start site, alternative splicing and utilization of alternative exons
and variable number of repeats complicate the nucleotide numbering and
here too a reference sequence needs to be established. The nomenclature, at least in the present state of the human genome development, needs to
be accurate, unambiguous, but flexible. The nucleotide change must
always be included in the original report; however other names, for
example based on amino acid changes, may be used. The genomic DNA-based
nomenclature was termed "systematic" by Beutler at al, while all other
mutation names were considered as "trivial" or "common" by these
authors.&nbsp;</p>

<p><b>A list of recommendations follows:</b></p>

<ul>
  <li>       For genomic DNA and cDNA , the A of the ATG of the initiator Met codon
        is nucleotide +1. There is no nucleotide zero (0). The nucleotide 5' to
        +1 is numbered -1. If there is more than one potential ATG, a reference
        consensus may be used. The numbering of nucleotides in the reference
        sequence in the databases should not be changed and will always be
        associated with the same (original) accession number.</li>
  <li>The use of lower case g for genomic or c for cDNA in front of the
        nucleotide number is recommended. To avoid confusion, a dot should
        separate these from the nucleotide number (g. or c. for genomic or cDNA
        respectively). The reference sequence Genbank number should also be
        included in the original publication/database submission.&nbsp;</li>
  <li>Nucleotide changes start with the nucleotide number and the
        change follows this number. 1996G>T denotes that at nucleotide 1996 of
        the reference sequence, G is replaced by a T.&nbsp;</li>
  <li>Deletions are designated by del after the nucleotide number.
        1996delT denotes the deletion of T at nt 1996. 1996-1998del denotes the
        deletion of 3 nts. Alternatively this mutation can be denoted as 1996-1998delTTC. For deletions in short tandem repeats the most 3' nt is
        arbitrarily assigned; eg. a TG deletion in the sequence AATGTGTGCC is
        designated 1996-1997delTG or 1996-1997del (where 1996 is the first T
        before C).&nbsp;</li>
  <li>Insertions are designated by ins after the nucleotide interval
        number. 1996-1997insT denotes that T was inserted in the interval
        between nts 1996 and 1997. For insertions in short repeats the most 3'
        nt interval is arbitrarily assigned; eg. a TG insertion in the sequence
        AATGTGTGCC is designated 1996-1997insTG (where 1996 is the last G of the
        short TG repeat).&nbsp;</li>
  <li>DNA polymorphic variants may be designated as alternative
        possibilities by an /. For example, 1996G/A denotes that in a given
        normal population there is either G or A at nt 1996. The system can
        accommodate short sequence repeat polymorphisms; eg 1996(GT)6-22. In
        this case, 1996 is the first nucleotide of the dinucleotide repeat.</li>
  <li>A unique identifier per mutation should be obtained. The OMIM
        unique identifier can be used, or database curators may assign such
        unique identifiers.&nbsp;</li>
  <li>Intron mutations when the full genomic sequence is not known can
        be designated by the intron (IVS) number, positive numbers starting from
        the G of the donor site invariant GT, negative numbers starting from the
        G of the acceptor site invariant AG. IVS4+1G>T denotes the G to T
        substitution at nt +1 of intron 4. IVS4-2A>C denotes the A to C
        substitution at nt -2 of intron 4. When the full length genomic sequence<br>
        is known, the mutation can also be simply designated&nbsp;by the nt number of the reference sequence.</li>
  <li>Two mutations in the same allele can be listed within brackets
        as follows: [1996G>T; 2001A>C]. This will also allow the (i) designation
        of mutations that are only deleterious when they occur in the same
        allele with additional nucleotide substitutions; (ii) designation of
        haplotypes of different alleles.&nbsp;</li>
  <li>For amino acid-based systems, the codon for the initiator
        Methionine is codon 1.</li>
  <li>The single letter amino acid code is recommended. However the
        three letter code is also acceptable.</li>
  <li>For amino acid nomenclature, the format is Y96S (Tyrosine at
        codon 96 substituted by Serine). The "wild type" amino acid is given
        before and the mutant amino acid after the codon number. Therefore there
        is no confusion for the significance of G,C,T and A in the
        nomenclature.&nbsp;</li>
  <li>Stop codons are designated by X. For example R96X (Arginine
        codon 96 substituted by a termination codon).&nbsp;</li>
  <li>Deletions of amino acids are designated as: T96del denotes that
        the codon 96 for Threonine is deleted.</li>
  <li>Insertions of amino acids are designated as: T96-97ins denotes
        that a codon for Threonine is inserted at the interval between codons 96
        and 97 of the reference sequence of the protein.</li>
  <li>Normal amino acid polymorphic variants may be designated as
        alternative possibilities by an /. For example, G/A96 denotes that in a
        given normal population there is either Glycine or Alanine at codon 96.</li>
  <li>The first report of a mutation in the literature should contain
        both a nucleotide and amino acid based name when appropriate.</li>
</ul>
<p>
No recommendations are made at this time for more complex mutations.
Detailed description for such mutations and nomenclature proposals can
be usually found in the original reference or by the unique identifier.
A second phase of recommendations will deal with such issues in the
future. In addition, the consequence of a mutation (frameshift,
particular splicing abnormality, exon skipping etc) is not included in
the mutation name. However, investigators that maintain mutation
databases are encouraged to include a field of mutation consequence or
mutation mechanism (if known) in their databases.<br>
<br>
This version of this document will be posted for 6 weeks starting
2nd December, 1996.&nbsp;<br>
<br>
After this date the recommendations will be published as guidelines for
investigators.<br>
<br>
Please forward to my email address (sea@medsun.unige.ch) your comments,
suggestions, criticisms.&nbsp;<br>
<br>
We would also invite you to co-sign this document (if you do not have
major disagreements with its content). Your participation and names will
appear in the published version of these recommendations. Please include
your institution, address, and email coordinates.&nbsp;</p>

<hr>

<p>Stylianos E. Antonarakis MD, DSc&nbsp;<br>
Professor of Medical Genetics&nbsp;<br>
University of Geneva Medical School&nbsp;<br>
9 avenue de Champel&nbsp;<br>
1211 Geneva&nbsp;<br>
SWITZERLAND<br>
<br>
Tel. +41-22-702.57.07<br>
Fax: +41-22-702.57.06<br>
E-mail: sea@medsun.unige.ch</p>

<font face="Verdana, Arial, Helvetica, sans-serif">

<hr>

<table width="100%" border="0" cellspacing="0" cellpadding="15" bgcolor="#ff9900">
  <tr>
    <td><p align="center"><font color="#FFFFFF" face="Verdana, Arial, Helvetica, sans-serif" size="-5"><a href="http://www.HGVS.org/copyright.html">Copyright &copy;</a> </font><font size="1"> HGVS 2007 All Rights Reserved<br>
    Website Created by Rania Horaitis, Nomenclature by J.T. Den Dunnen - <a href="http://www.HGVS.org/disclaimer.html">Disclaimer</a> </font></td>
  </tr>
</table>
</font> 
</body>
</html>
