---
parent: DNA
title: repeated sequences
---


    <p>A frequently occuring sequence change is the variability of repeated
      sequences. Within this category we discriminate both small sequences
      (mono-, di-, tri-, etc nucleotide repeats) as well as the much larger
      ones. Such changes are described using the format <em><strong>"position-first-repeat-unit_[number]</strong></em>"
      (e.g. g.123_124[4]) where <em><strong>position-first-repeat-unit</strong>
        gives the location of the first unit</em> of the variable sequence
      repeat and <em><strong>[number]</strong> the number of units present</em>
      in the allele described. </p>
    <ul>
      <li>the first unit of the repeat is preferably described based on
        position, like. g.123_124. <em>For short/simple repeats it is
          acceptable to include the content of the repeated unit, using the
          format "position-first-nucleotide-repeat_content" like g.123TG[4].</em>
        <strong>Do not use</strong> a mix of these descriptions like
        g.123_124TG[4]. This contains redundant information (123_124 and TG)
        with the danger of being in conflict. <br>
        <em><span style="font-weight: bold;">NOTE:</span> including the content
          of the sequence involved quickly gives descriptions which become too
          lengthy.</em></li>
      <li>uncertainties regarding the number of repeated copies are given
        between brackets, like c.-128GGC[(600_800)]. </li>
      <li><span style="font-weight: bold; font-style: italic;"><i><img src="file:///Users/Boomheide/Files/WWW/HGVS/new.gif"
              border="0"
              height="12"
              width="31"></i></span>the
        repeated sequences may have complex structures, consisting of a mix of
        several repeated sequences directly following each other. When
        sequenced, one should describe it including the individual repeated
        elements, like g.456TG[4]TA[9]TG[3] (or
        g.456_465[4]466_489[9]490_499[3]). When not sequenced, but <span style="font-weight: bold; font-style: italic;">based
          on fragment size</span>, it should be described like g.456_465[16].</li>
      <li>the same format can be used to describe the presence of multiple
        copies (triplication, quadruplication, etc.) of larger sequences, e.g.
        exons. Square brackets (<em><strong>"[ ]"</strong></em>) should only be
        used when their is experimental evidence that the additional copies are
        in tandem on the same chromosome.<br>
        <em><strong>NOTE:</strong> the description<strong> "dup"</strong> (<a href="standards.html#define">see
            Standards</a>) may <strong>by definition</strong> only be used when
          the additional copy is directly 3'-flanking of the original copy
          (tandem duplication). In many cases there will be <strong>no
            experimental proof</strong>, the additional copy may be anywhere in
          the genome (i.e. inserted / transposed). It has been suggested that,
          unless there is experimental evidence, exonic duplications should be
          described using the format, i.e. c.88-?_301+?<b>(2)</b>.</em></li>
    </ul>
    <h4>Examples</h4>
    <ul>
      <li>g.123_124[4] (<em>alternatively g.123TG[4]</em>) describes a sequence
        variable in the number of TG repeats where the first unit is present at
        position g.123_124 in the genomic reference sequence and the allele
        described contains 4 units.</li>
      <li>c.1210-12T[5_9] describes a famous variable stretch of 5 to 9
        T-residues in intron 9 of the CFTR gene. The most commonly used CFTR
        coding DNA reference sequence contains a stretch of 7 T's. It is
        recommended to describe individual alleles differing from this reference
        sequence as c.1210-12T[5] (<i>not c.1210-7_1210-6delTT</i>) or
        c.1210-12T[9] (<i>not c.1210-7_1210-6dupTT</i>).<br>
        <i><b>NOTE: </b>the repeat should not be described as c.1210-6T[5_9]</i></li>
      <li>c.123+74TG[3_6] (<i>alternatively c.123+74_123+75[3_6]</i>) indicates
        that a TG di-nucleotide repeat is present, starting at nucleotide 74 in
        the intron following cDNA nucelotide c.123, which is found varying from
        3 to 6 copies in the population
        <ul>
          <li><i>c.123+74TG[4];[5] denotes that a person carries a TG
              di-nucleotide repeat of length 4 on one chromosome and of length 5
              on the other chromosome</i></li>
        </ul>
      </li>
      <li>in literature the Fragile-X tri-nucleotide repeat is known as the
        CGG-repeat, but based on the coding DNA reference sequence (GenBank
        NM_002024.4) and the rule that for all descriptions the <span style="font-weight: bold; font-style: italic;">most
          3' position possible</span> should be arbitrarily assigned the repeat
        has to be described as a GGC-repeat. In addition the repeat is
        interrupted by GGA triplets (see e.g. <a style="font-style: italic;" href="http://www.ncbi.nlm.nih.gov/pubmed/8634688">Eichler
          1995</a>) making it a complex repeat which can not be accurately
        described based on sizing only. The sequence represented by the FMR1
        coding DNA Reference Sequence (GenBank <a href="http://www.ncbi.nlm.nih.gov/nucleotide/NM_002024.5">NM_002024.5</a>)
        is c.-128GGC[9]GGA[1]GGC[10].<br>
        <span style="font-weight: bold; font-style: italic;">NOTE:</span> based
        on coding DNA reference sequence <a href="http://www.ncbi.nlm.nih.gov/nucleotide/NM_002024.3">NM_002024.3</a>
        this variant is described as c.-158GGC[9]GGA[1]GGC[9]GGA[1]GGC[10]. To
        prevent such differences the recommendation is to use the stable LRG
        reference sequence (Locus Reference Genomic sequence, <span style="text-decoration: underline;"><span
            style="font-style: italic;"></span></span><a
          href="http://www.genomemedicine.com/content/pdf/gm145.pdf"><i>Dalgleish
            et al. 2010</i></a>); for FMR1 LRG_762t1:c.-128GGC[9]GGA[1]GGC[10].
        <ul>
          <li>c.-128_-126[79] describes the presence of an allele with an
            extended GGC-repeat of <i><b>exactly 79 units</b></i>.<br>
            <span style="font-weight: bold; font-style: italic;"><i><img src="file:///Users/Boomheide/Files/WWW/HGVS/new.gif"
                  border="0"
                  height="12"
                  width="31"></i>NOTE:</span>
            the description c.-128GGC[79] can not be used since the repeat is
            probably interrupted by one or more GGA-triplets</li>
          <li>c.-128_-126[(600_800)] describes the presence of an extended
            GGC-repeat with an estimated length between 600 and 800 copies<br>
            <b><i><i><img src="file:///Users/Boomheide/Files/WWW/HGVS/new.gif" border="0"
                    height="12"
                    width="31"></i>NOTE:</i></b>
            brackets are used to indicate uncertainties (<a href="file:///Users/Boomheide/Files/WWW/HGVS/uncertain.html"><i>see
                Uncertainties</i></a>), the description c.-128GGC[(600_800)] can
            not be used since the repeat is probably interrupted by one or more
            GGA-triplets </li>
          <li>c.-128_-126[(1000)] describes the presence of an extended
            GGC-repeat of about 1000 units.<br>
            <b><i><i><img src="file:///Users/Boomheide/Files/WWW/HGVS/new.gif" border="0"
                    height="12"
                    width="31"></i>NOTE:</i></b>
            brackets are used to indicate uncertainties (<a href="file:///Users/Boomheide/Files/WWW/HGVS/uncertain.html"><i>see
                Uncertainties</i></a>), the description c.-128GGC[(1000)] can
            not be used since the repeat is probably interrupted by one or more
            GGA-triplets</li>
        </ul>
      </li>
      <li><a name="tri"></a><i><img src="new.gif" border="0" height="12" width="31"></i>c.1032-?_1357+?(3)&nbsp;
        denotes the presence of two additional copies of an exon, starting at an
        unknown position in the flanking upstream intron (upstream of coding DNA
        nucleotide 1032) and ending at an unknown position in the flanking
        downstream intron (downstream of coding DNA nucleotide 1357). The
        description, using "(3)" indicates there is no proof that the three
        copies are in tandem<i> (</i><em><a href="uncertain.html">see
            Uncertainties</a></em>)</li>
      <li><i><img src="new.gif" border="0" height="12" width="31"></i>c.1032-?_1357+?[3]&nbsp;
        denotes a <b>direct triplication</b> of an exon, starting at an unknown
        position in the flanking upstream intron (upstream of coding DNA
        nucleotide 1032) and ending at an unknown position in the flanking
        downstram intron (downstream of coding DNA nucleotide 1357)<br>
        <i><b>NOTE:</b> the use of tri (for triplication), qua (for
          quadruplication), etc. is not recommended</i> (<a href="disc.html#tri"><i>see
            Discussion</i></a>) </li>
      <li>g.1209_4523[12_45] denotes that a 3.3 kb repeat sequence of which the
        first copy is present in the genomic reference sequence from nucleotides
        1209 to 4523 can be found repeated 12 to 45 times in the population
        <ul>
          <li><i>g.1209_4523[14];[23] denotes that a person carries a 3.3 kb
              repeat with 14 copies on one chromosome and of 23 copies on the
              other chromosome<br>
            </i></li>
        </ul>
      </li>
    </ul>