---
layout: default-md
title: Basics
order: 1
---

# {{ page.title }}

* * * 

The **HGVS nomenclature** is authorised by the [Human Genome Variation Society (HGVS)](http://www.hgvs.org){:target="\_blank"}, the [Human Variome Project (HVP)](http://www.humanvariomeproject.org){:target="\_blank"} and the [HUman Genome Organisation (HUGO)](http://www.hugo-international.org){:target="\_blank"}. Activities regarding the nomenclature go through a committee, the **HGVS nomenclature Sequence Variant Description Working Group (SVD-WG)**, with administrative support of the HVP office ([see HVP website](http://www.humanvariomeproject.org/sdp/wg04-sequence-variant-description-committee.html){:target="\_blank"}).

Discussions regarding the uniform and unequivocal description of sequence variants in DNA and protein sequences (mutations, polymorphisms) were initiated by two papers published in 1993 ([Beaudet AL & Tsui LC](http://onlinelibrary.wiley.com/doi/10.1002/humu.1380020402/abstract){:target="\_blank"} and [Beutler E](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682427/pdf/ajhg00054-0240.pdf){:target="\_blank"}). The ideas presented were widely discussed, modified, extended ([_see History_](/history)) and ultimately resulted in the **HGVS nomenclature**, the recommendations for the description of sequence variants, based on a paper published in 2000 ([den Dunnen, JT and Antonarakis, SE](http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART){:target="\_blank"}). Initially the recommendations did not cover all types of variants and more complex changes and the nomenclature website was started to publish the latest updates. The last publication covering the recommendations was published in 2016 ([den Dunnen, JT et al. 2016](http://onlinelibrary.wiley.com/doi/10.1002/humu.22981/pdf){:target="\_blank"}).

This nomenclature website was established to give an up-to-date overview of all recommendations, the changes made and the extensions approved. In addition the web site provides background information, examples of descriptions based on the nomenclature and link to support tools and educational material. These pages can be used as a guide to describe any sequence variant identified and should help to get a uniformly accepted standard. The chair of the SVD-WG takes care of this website, answers questions, handles requests to change or extend the recommendations, prepares proposals for community consultation, publishes new versions of the standards and assigns HGVS nomenclature version numbers.

### Questions?

Questions about the HGVS nomenclature should be addressed to **"Varnomen @ variome.org"** (remove spaces). When having a specific question, please do not forget to mention the **reference sequence(s)** you use!

### Changes

Any proposal made by the SVD-WG will be published on this website and opened for a 2-month period of **Community Consultation**. During this period everybody interested is asked to study the proposal and send comments, positive or negative, to the committee. Comments to proposals should be addressed to **"Varnomen @ variome.org"** (remove spaces), Subject: SVD-WGxxx (xxx the proposal number, e.g. SVD-WG001). After two months the committee will collect all responses, evaluate them and take a decision. Proposals might be approved, rejected or modified for a new round of consultation. All decisions will be published on this website. To make sure that you receive notification of any proposal published for Community Consultation, please register at the HVP office (<u>E-mail</u> info @ Variome.org, remove spaces), <u>Subject</u>: SVD-WG discussion list). For proposals open for Community Consultation and issues currently discussed, [_see Open Issues_](/recommendations/open-issues/).

### Versioning

The recommendations for the description of sequence variants are designed to be _**stable**_, _**meaningful**_, _**memorable**_ and _**unequivocal**_. Still, every now and then small modifications will be required to remove inconsistencies and/or to clarify confusing conventions. In addition, the recommendations may be extended to resolve cases that were hitherto not covered. Since 2015, **any change** in the recommendations receive a new **version number**. The version number is based on the date of the change and has the format: **HGVS nomenclature _Version 15.11_**, for the version accepted in 2015 ("**15**"), November ("**11**"). See the [_Version list_](/versioning/) for a complete historic overview. 

The current HGVS version number is shown in the top right corner of this web site ("_**Version xx.xx**_"). Note that the version number remains as is when only a typing error is corrected, an example added, an explanation clarified, a question answered, etc. Users can specify up to what point they follow the HGVS recommendations by mentioning the version number in their report.

* * *

## Terminology 

Considerable effort has been invested to use only clearly defined terms. For details see our [_Glossary_](/bg-material/glossary/).

### Mutation and polymorphism

In some disciplines the term **"mutation"** is used to indicate "_a change_" while in other disciplines it is used to indicate "_a disease-causing change_". Similarly, the term **"polymorphism"** is used both to indicate "_a non disease-causing change_" or "_a change found at a frequency of 1% or higher in a population_". To prevent this confusion we do not use the terms mutation and polymorphism (including SNP or Single Nucleotide Polymorphism) but neutral terms like **"variant"**, **"change"** and **"allelic variant"**. Vol.19(1) of Human Mutation (2002) contains several contributions discussing the issues and shows that the term **"mutation"** has developed a negative connotation (see [Cotton RGH - p.2](http://onlinelibrary.wiley.com/doi/10.1002/humu.10029/pdf){:target="\_blank"}, [Condit CM et al. - p.69](http://onlinelibrary.wiley.com/doi/10.1002/humu.10023/pdf){:target="\_blank"} and [Marshall JH - p.76](http://onlinelibrary.wiley.com/doi/10.1002/humu.10021/pdf){:target="\_blank"}). Who would like to carry a mutation and thus be a **"mutant"**.  Current guidelines of authorative organisations now also recommend to use neutral terms like **"variant"** and **"change"** only (see [Richards 2015, Genet.Med. 17:405-424](http://www.nature.com/gim/journal/v17/n5/pdf/gim201530a.pdf){:target="\_blank"}).

In cancer genetics the terms "mutation" and "mutation load" are often used. While, compared to the reference sequence, changes in the genome of the individual should be called "variants", additional changes (de novo somatic variants) in the cancer tissue are called "mutations". In such cases, "mutation load" and "tumor mutation burden" are acceptable terms, but only for variants which are proven to be de novo/somatic and not present in the germline of the individual.

### Pathogenic

Another confusing term used frequently is "a **pathogenic** variant". A variant in itself is not "pathogenic", whether it can be causally related to a phenotype observed in a patient is determined by other factors. While a non-expert concludes the variant described "_**causes disease**_", the expert probably means "_**causes disease when in a specific context**_", e.g:

*   causes disease when found in a male (X-linked recessive disorder)
*   causes disease when combined with a change on the other allele (autosomal recessive)
*   causes disease when inherited from the father (imprinting disorder)

Another problem emerges when variants need to be classified that determine non-disease phenotypes like lactose tolerance/persistence, bitter tasting or hair and skin colour. What is pathogenic in the MC1R gene: dark, blond, or red hair?  What ends up in the SLC24A5 database as the pathogenic skin color: light or dark skin?

To prevent confusion HGVS recommends not to use the term "_pathogenic_". For variant classification a good alternative is a specific but neutral term like "**affects function**". This properly describes what one actually means, the variant affects the normal function of the gene/protein (in whatever way). It also solves the issue of what term to use for variants associated with non-disease phenotypes like skin/hair/eye colour. To indicate that in an individual variant(s) were found that can explain the disease phenotype an alternative for "_pathogenic_" is to use "**disease-associated variant**".

Another option is to give for each variant both a "**functional**" and a "**clinical**" classification. A five category functional classifications system could use; _affects function,  probably affects function,  unknown (or effect on function not known),  probably does not affect function (or probably no functional effect),  does not affect function (or no functional effect)_. A five category clinical classification system could use known terms like "pathogenic", "unknown" and "benign" with additions as mentioned above, e.g. "disease-associated (recessive)", "phenotype-associated (dominant)", "disease-associated (paternal)", etc.
