---
layout: default-md
title: Basics
order: 1
---

# {{ page.title }}

* * * 

Discussions regarding the uniform and unequivocal description of sequence variants in DNA and protein sequences (mutations, polymorphisms) were initiated by two papers published in 1993 ([_Beaudet AL & Tsui LC_](http://onlinelibrary.wiley.com/doi/10.1002/humu.1380020402/abstract) and [_Beutler E_](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682427/pdf/ajhg00054-0240.pdf)). The ideas presented were widely discussed, modified, extended ([_see History_](/history)) and ultimately resulted in the **HGVS nomenclature**, the recommendations for the description of sequence variants, based on a paper published in 2000 ([den Dunnen, JT and Antonarakis, SE](http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART)). Initially these recommendations did not extensively cover all types of variants and more complex changes. This website was established to give an up-to-date overview of all recommendations, the changes made and the extensions approved. In addition the web site provides background information, examples of descriptions based on the nomenclature and link to support tools and educational material. These pages can be used as a guide to describe any sequence variant identified and should help to get a uniformly accepted standard.

The **HGVS nomenclature** is authorised by the [Human Genome Variation Society (HGVS)](http://www.hgvs.org), the [Human Variome Project (HVP)](http://www.humanvariomeproject.org) and the [HUman Genome Organisation (HUGO)](http://www.hugo-international.org). Activities regarding the nomenclature go through a committee, the **Sequence Variant Description Working Group (SVD-WG)** established by these 3 organisations. The SVD-WG operates according to a charter defining their activities [see HVP website](http://www.humanvariomeproject.org/sdp/wg04-sequence-variant-description-committee.html), with administrative support of the HVP office.

The SVD-WG takes care of this website, answers questions, handles requests to change or extend the recommendations, prepares proposals for community consultation, publishes new versions of the standards and assigns HGVS nomenclature version numbers. Any proposal made by the SVD-WG will be published on this website and opened for a 2-month period of **Community Consultation**. During this period everybody interested is asked to study the proposal and send comments, positive or negative, to the committee. Comments to proposals should be addressed to **"Varnomen @ variome.org"**, Subject: SVD-WGxxx (xxx the proposal number, e.g. SVD-WG001). After two months the committee will collect all responses, evaluate them and make a decision. Proposals might be approved, rejected or modified for a new round of consultation. All decisions will be published on this website. To make sure that you receive notification of any proposal published for Community Consultation, please register at the HVP office (mail to Timothy Smith (Tim @ Variome.org), subject: SVD-WG discussion list).

* * *

## Versioning

The recommendations for the description of sequence variants are designed to be **_stable_**, **_meaningful_**, **_memorable_** and **_unequivocal_**. Still, every now and then small modifications will be required to remove inconsistencies and/or to clarify confusing conventions. In addition, the recommendations may be extended to resolve cases that were hitherto not covered. To allow users to specify up to what point they follow HGVS nomenclature, version numbers will be assigned.

Since 2015, **any change** in the recommendations receives a new **version number**. The version number will be based on the date of the change. Both in the [version list](../../versioning), and on the page containing the change, the version number assigned will be clearly marked. The version number will have the format: **HGVS nomenclature _Version 15.11_**, for the version accepted in 2015 ("**15**"), November ("**11**").

The current HGVS version number is shown in the top right corner of this web site ("_**Version xx.xx**_"). Note that when a typing error is corrected, an example added, an explanation further clarified, a question answered, etc., the recommendations not actually change and the version number therefore will not change.

*	**Version 0**
:	On the page "[History regarding the description of sequence variants](/history)" we give an overview of all publications on the description of sequence variants. These papers can be considered as pre-versions of the first recommendations, a version 0.
*	**Version 1**
:	The 2000 publication of Den Dunnen JT and Antonarakis SE [_Mutation nomenclature extensions and suggestions to describe complex mutations: a discussion. Hum.Mutat. 15:7-12](http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART)) contain a more formal set of recommendations an are considered as version 1.
*	**Version 15.11**
:	HGVS nomenclature version 15.11 is described in [Den Dunnen et al. (2016). HGVS recommendations for the description of sequence variants: 2016 update. Hum.Mutat. _in press_](XXX). For the most significant changes between this version and version 1.0 [_see Version list_](../../versioning).

* * *

## Terminology 

Considerable effort has been invested to use for the HGVS nomenclature (and on these pages) only clearly defined terms. A Glossary with an extensive list of definitions is _**in preparation**_. Terms that might cause confusion will not be used. Below a short explanation for some of these.

#### Mutation and polymorphism

In some disciplines the term**"mutation"** is used to indicate "_a change_" while in other disciplines it is used to indicate "_a disease-causing change_". Similarly, the term **"polymorphism"** is used both to indicate "_a non disease-causing change_" or "_a change found at a frequency of 1% or higher in the population_". To prevent this confusion we do not use the terms mutation and polymorphism (including SNP or Single Nucleotide Polymorphism) but use neutral terms like **"sequence variant"**, **"alteration"** and **"allelic variant"**. The Vol.19(1) issue of Human Mutation (2002) contains several contributions discussing these issues as well as the fact that the term **"mutation"** has developed a negative connotation (see [Cotton RGH - p.2](http://onlinelibrary.wiley.com/doi/10.1002/humu.10029/pdf), [Condit CM et al. - p.69](http://onlinelibrary.wiley.com/doi/10.1002/humu.10023/pdf) and [Marshall JH - p.76](http://onlinelibrary.wiley.com/doi/10.1002/humu.10021/pdf)). Current guidelines of authorative organisations now also recommend to use neutral terms like "**variant**" and "**change**" only (see [Richards 2015, Genet.Med. 17:405-424](http://www.nature.com/gim/journal/v17/n5/pdf/gim201530a.pdf)).

#### Pathogenic

Another confusing term used frequently is "a **pathogenic** variant". While a non-expert concludes the variant described "_causes disease_", the expert probably means "_causes disease when in a specific context_"

*   causes disease when found in a male (X-linked recessive disorder)
*   causes disease when combined with a similar chance in the other allele (autosomal recessive)
*   causes disease when inherited from the father (imprinted)

To prevent confusion it therefore seems best not to use the term "_pathogenic_". A good alternative seems a neutral term like "**affects function**". In fact this properly describes what one actually means, the variant affects the normal function of the gene/protein (in whatever way). This also solves the issue of what term to use for non-disease phenotypes like skin/hair/eye colour or blood group. In such cases it is problematic to choose the phenotype to call "_normal_" or "_pathogenic_". Using "**affects function**" is clear and effective. To classify variants people use most frequently 5 categories. Based on affects function these could be; _affects funtcion,  probably affects function,  unknown,  probably does not affect function (or probably no functional effect),  does not affect function (no functional effect)_. Variants for which a functional effect is unknown can together be called "**variants of unknown significance**" (**VUS**).


[Slide Presentation - pdf](/assets/HGVS-basics2014.pdf)