---
layout: default
title: Glossary
order: 7
---
## {{ page.title }}

---

### In preparation

Please note that this Glossary is **work in progress**. Do you encounter missing terms or want to suggest definitions, please let us know.

---

*   3'rule
    :    for all descriptions the most 3â€™ position possible of the reference sequence is arbitrarily assigned to have been changed. When ATTTG changes to ATTG HGVS describes this as a change of the T at position 4 (not the T at position 2 or 3) 
*   allele
    :   variant forms of the same gene ([MESH](http://www.ncbi.nlm.nih.gov/mesh?term=allele){:target="\_blank"})
    :   **HGVS**: a series of variants on one chromosome.
    :   descriptions _see Recommendations_ [_DNA_](/recommendations/DNA/variant/alleles/), [_RNA_](/recommendations/RNA/variant/alleles/) or [_protein_](/recommendations/protein/variant/alleles/).

*   amino acid
    :   a letter from the protein code ([_see Standards_](/bg-material/standards/)).

*   cap site
    :   first nucleotide of a transcript (5' end) to which a specially altered nucleotide is added.
    
*   cDNA
    :   cDNA, "copy DNA" or "complementary DNA", is the DNA copy of a single stranded RNA molecule synthesized using the enzyme reverse transcriptase ([Wikipedia](https://en.wikipedia.org/wiki/Complementary_DNA){:target="\_blank"}, [MESH](https://www.ncbi.nlm.nih.gov/mesh/68018076){:target="\_blank"}).
    :   _**NOTE:**_ cDNA is not the same as "coding DNA" (see below).

*   CDS
    :   coding DNA sequence.
   
*   cis
    :   two variants are **"in cis"** when they are on the same allele (DNA molecule, chromosome).
    
*   coding DNA
    :   the segments of a genome or segment of a transcript (RNA molecule) which codes for a protein.
    :   _**NOTE:**_ a [_coding DNA reference sequence_](/bg-material/refseq/#DNAc) is **not** a cDNA sequence (see above)

*   chimerism
    :   the occurrence in one individual of two or more cell populations, derived from different zygotes, with different sequences (based on [MESH](http://www.ncbi.nlm.nih.gov/mesh?term=chimerism){:target="\_blank"}). Opposite of mosaicism.
    :   descriptions [_see General/Charcters used_](/recommendations/general#characters).

*   complex
    :   **HGVS**: a sequence change where, compared to a reference sequence, a range of changes occur that can not be described as one of the basic variant types (substitution, deletion, duplication, insertion, conversion, inversion, deletion-insertion, or repeated sequence).

*   compound heterozygote
    :   used in cases of _autosomal recessive_ disease where the disease-causing variants on both alleles at a given locus are **not identical** (opposite of _homozygous_)
    
*   conversion
    :   **HGVS**-DNA: a sequence change where, compared to a reference sequence, a range of nucleotides are replaced by a sequence from elsewhere in the genome.
    :   descriptions _see Recommendations_ [_DNA_](/recommendations/DNA/variant/conversion/) or  [_RNA_](/recommendations/RNA/variant/conversion/).

*   deletion
    *   one or more letters of the DNA code are missing (deleted). A deletion is indicated using a **"del"**
    *   **HGVS**-DNA: a sequence change where, compared to a reference sequence, one or more nucleotides are not present (deleted).
    :   descriptions _see Recommendations_ [_DNA_](/recommendations/DNA/variant/deletion/), [_RNA_](/recommendations/RNA/variant/deletion/) or [_protein_](/recommendations/protein/variant/deletion/).

*   deletion-insertion (delins)
    *   one or more letters in the DNA code are missing and replaced by several new letters
    *   **HGVS**-DNA: a sequence change where, compared to a reference sequence, one or more nucleotides are replaced by one or more other nucleotides and which is not a substitution, inversion or conversion..
    :   descriptions _see Recommendations_ [_DNA_](/recommendations/DNA/variant/indel/), [_RNA_](/recommendations/RNA/variant/indel/) or [_protein_](/recommendations/protein/variant/indel/).

*   duplication
    *   one or more letters of the DNA code are present twice (doubled, duplicated)
    *   **HGVS**-DNA: a sequence change where, compared to a reference sequence, a copy of one or more nucleotides are inserted directly 3' of the original copy of that sequence.
    :    _**NOTE:**_ diagnostic assays (like MLPA) usually detect _**an additional copy**_ of a specific sequence. Whether the additional copy is a duplication or an insertion remains to be determined.
    :   descriptions _see Recommendations_ [_DNA_](/recommendations/DNA/variant/duplication/), [_RNA_](/recommendations/RNA/variant/duplication/) or [_protein_](/recommendations/protein/variant/duplication/).

*   exon
    :   any nucleotide sequence within a gene which, during maturation of the RNA transcript, is not removed by a process called RNA splicing ([Wikipedia](https://en.wikipedia.org/wiki/Exon){:target="\_blank"}, [MESH](https://www.ncbi.nlm.nih.gov/mesh/68005091){:target="\_blank"}). Every exon, except the first and lat exon, is flanked by two **introns**.
    
*   extension
    :   a sequence change extending the normal amino acid sequence at the N- or C-terminal end with one or more amino acids ([_protein_](/recommendations/protein/variant/extension/))

*   frame shift
    :   a sequence change between the translation initiation (start) and termination (stop) codon where, compared to a reference sequence, translation shifts to another reading frame ([_protein_](/recommendations/protein/variant/frameshift/))

*   hemizygous
    :   an individual having **only one allele** at a given locus, either because the allele is absent (X and Y chromosome in males) or lost (deleted) (based on [MESH](http://www.ncbi.nlm.nih.gov/mesh?term=hemizygous){:target="\_blank"}).
    
*   heterozygous
    :   an individual in which both alleles at a given locus are **not identical** (based on [MESH](http://www.ncbi.nlm.nih.gov/mesh?term=heterozygous){:target="\_blank"}).
    
*   homozygous
    :   an individual in which both alleles at a given locus are **identical** ([MESH](http://www.ncbi.nlm.nih.gov/mesh?term=homozygous){:target="\_blank"}).
    
*   indel
    :   **HGVS**: confusing term, not used
        :   _**sometimes**_: a sequence change where, compared to a reference sequence, one or more nucleotides are replaced by one or more other nucleotides
        :   _**sometimes**_: a variant which is a deletion or an insertion.
        :   _**sometimes**_: (evolutionary biology) a type of variant in which a specific nucleotide sequence is present (insertion) or absent (deletion). 
        :   [MESH](http://www.ncbi.nlm.nih.gov/mesh?term=indel){:target="\_blank"}: a length difference between two alleles where it is unknowable if the difference was originally caused by a sequence insertion or a sequence deletion 

*   insertion
    *   one or more letters in the DNA, RNA or amino acid code are new (have been inserted)
    *   **HGVS**-DNA: a sequence change where, compared to the reference sequence, one or more residues are inserted and where the insertion is not a copy of a sequence immediately upstream.
    :   descriptions _see Recommendations_ [_DNA_](/recommendations/DNA/variant/insertion/), [_RNA_](/recommendations/RNA/variant/insertion/) or [_protein_](/recommendations/protein/variant/insertion/).

*   intron
    :   any nucleotide sequence within a gene which, during maturation of the RNA transcript, is removed by a process called RNA splicing ([Wikipedia](https://en.wikipedia.org/wiki/Intron){:target="\_blank"}, [MESH](https://www.ncbi.nlm.nih.gov/mesh/68007438){:target="\_blank"}). Every intron is flanked by two **exons**.

*   inversion
    :   **HGVS**-DNA: a sequence change where, compared to a reference sequence, more than one nucleotide replacing the original sequence are the reverse complement of the original sequence.
    :   descriptions _see Recommendations_ [_DNA_](/recommendations/DNA/variant/inversion/) or [_RNA_](/recommendations/RNA/variant/inversion/).
    
*   ISCN
    :   International System for Cytogenetic Nomenclature [_(see ISCN)_](/bg-material/consultation/ISCN), covering the description of numerical and structural chromosomal changes detected using microscopic and cytogenetic techniques.
    :   descriptions _see Recommendations_ [_DNA - Complex (HGVS<>ISCN)_](/recommendations/DNA/variant/complex/).

*   Kozak sequence
    :   a consensus sequence, including the ATG translation initiation codon, playing a role in the initiation of translation

*   LOH
    :   Loss of Heterozygosity (LOH) is a term originally derived from the analysis of tumor samples where, as a consequence of a somatic change, a cell that had originally two different alleles **looses one allele**. The LOH can be caused by different molecular mechanism, including the deletion of the allele, a gene conversion or uniparental disomy.
    :   _**NOTE**:_ the definition given by [MESH](http://www.ncbi.nlm.nih.gov/mesh/?term=loss+of+heterozygosity){:target="\_blank"}, i.e. the loss of one allele at a specific locus caused by a deletion, is therefore not correct
    :   _**NOTE**:_ the term LOH should thus **not** be used to indicate a **homozygous** region, i.e. a region where both chromosomes have the same sequence.

*   loss of heterozygosity
    :   see LOH. 

*   missense
    *   a variant in which a codon is changed to one directing the incorporation of a different amino acid (based on [MESH](https://www.ncbi.nlm.nih.gov/mesh/?term=missense){:target="\_blank"}).
    *   **HGVS**: a variant in a protein sequence where compared to the reference sequence one amino acid is replaced by another amino acid.

*   mosaicism
    :   the occurrence in one individual of two or more cell populations, derived from a single zygote, with different sequences (based on [MESH](http://www.ncbi.nlm.nih.gov/mesh?term=mosaicism){:target="\_blank"}). Opposite of chimerism.
    :   descriptions [_see General/Characters used_](/recommendations/general#characters).
    
*   mutation
    *   **HGVS**: confusing term, do not use, use **variant** ([_see Basics_](/bg-material/basics/))
    *   _**biology**_: a change in the sequence
    *   _**medicine**_: a sequence variant **associated** with a disease phenotype

*   nonsense
    *   a variant that changed an amino acid-specifying codon to a stop codon (termination codon, based on [MESH](https://www.ncbi.nlm.nih.gov/mesh/?term=nonsense){:target="\_blank"}).
    *   **HGVS**: a variant in a protein sequence where compared to the reference sequence an amino acid is replaced by a translational stop codon (termination codon).

*   nucleotide
    :   a letter fron the DNA code, either A, C, G, or T ([_see Standards_](/bg-material/standards/))

*   polyA addition site
    :   the 3' end of a precursor messenger RNA (pre-mRNA) transcript that is cleaved and to which subsequently a tail of A nucleotides is added (the polyA-tail)

*   polyA signal
    :   a sequence in the 3' UTR of a transcript signalling the downstream cleavage and addition of a polyA tail 
    
<a name="polymorphism"></a>
*   polymorphism
    *   **HGVS**: confusing term, do not use, use **variant** ([_see Basics_](/bg-material/basics/))
    *   _**biology**_: a sequence variant present in the population at a frequency of 1% or higher
    *   _**medicine**_: a sequence variant **not associated** with a disease phenotype

*   quadruplication
    :   a sequence change where, compared to a reference sequence, three copies of a sequence are inserted directly 3' of the original copy of that sequence (_see Recommendations_ [_DNA_](/recommendations/DNA/variant/duplication/)).

*   quintuplication
    :   a sequence change where, compared to a reference sequence, four copies of a sequence are inserted directly 3' of the original copy of that sequence (_see Recommendations_ [_DNA_](/recommendations/DNA/variant/duplication/)).

*   repeated sequence
    :   **HGVS**: a sequence where, compared to a reference sequence, a segment of one or more nucleotides (the repeat unit) is present several times, one after the other

*   silent
    *   a variant in a DNA sequence that does not change the amino acid sequence of the encoded protein (based on [MESH](https://www.ncbi.nlm.nih.gov/mesh/?term=silent){:target="\_blank"}).
    *   **HGVS**: an amino acid residue in a protein sequence where compared to the reference sequence the DNA sequence changed but not the encoded amino acid.
    
*   SNP
    :   Single Nucleotide Polymorphism (SNP). Note the preferred term is SNV (Single Nucleotide Variant), [_see polymorphism_](#polymorphism)

*   SNV
    :   Single Nucleotide Variant (SNV). Term preferred over SNP.

*   splice acceptor site (SA)
    :   the 3' splice site, at the end of the intron/start of the exon
    
*   splice donor site (SD)
    :   the 5' splice site, at the end of the exon/start of the intron
    
*   splice site
    :   the site in a precursor messenger RNA (pre-mRNA) transcript that is cleaved to remove the intron.

*   splicing
    :   the process removing specific segments (the inrons) of a precursor messenger RNA (pre-mRNA) transcript. When an intron is removed the flanking RNA segments (the exons) are joined together (ligated)
    
*   substitution
    *   one letter of the DNA, RNA or amino acid code is replaced (substituted) by one other letter 
    *   **HGVS**-DNA: a sequence change where, compared to a reference sequence, one residue is replaced by one other residue.
    :   descriptions _see Recommendations_ [_DNA_](/recommendations/DNA/variant/substitution/), [_RNA_](/recommendations/RNA/variant/substitution/) or [_protein_](/recommendations/protein/variant/substitution/).
   
*   trans
    :   two variants are **"in trans"** when they are on different alleles (DNA molecules, chromosomes).
    
*   translocation
    *   a chromosome abnormality characterized by chromosome breakage and transfer of the broken-off portion to a non-homologous chromosome (based on [MESH](https://www.ncbi.nlm.nih.gov/mesh/?term=translocation){:target="\_blank"})
    *   **HGVS**: a sequence change where, compared to a reference sequence, from a specific nucleotide position (the break point) all nucleotides upstream derive from another chromosome then those down stream
    :   _**NOTE**:_ a translocation occurs when two chromosomes break and the fragments rejoin with the non-homologous chromosome. A full description of a (reciprocal) translocation consists of 2 parts, one describing the first junction, the second describing the other junction (e.g. the chromosome 4;X junction and the chromosome X;4 junction)
    *   translocation, balanced
        :   a translocation with an even exchange of DNA sequences and no segments deleted or duplicated
    *   translocation, un balanced
        :   a translocation with an uneven exchange of DNA sequences and segments being deleted or duplicated

*   transposition
    :   a sequence change where compared to a reference sequence a range of nucleotides moves from one position to another position, i.e. a deletion at one postion combined with the insertion of the deleted sequence at another position.

*   triplication
    :   a sequence change where, compared to a reference sequence, two copies of a sequence are inserted directly 3' of the original copy of that sequence (_see Recommendations_ [_DNA_](/recommendations/DNA/variant/duplication/)).

*   trisomy
    : the presence of a third chromosome of any one type in an otherwise diploid cell ([MESH](http://www.ncbi.nlm.nih.gov/mesh/?term=trisomy){:target="\_blank"}).

*   UTR
    :   UnTranslated Region (UTR), the segments of of a protein coding RNA molecule that is not translated.
    :   5'UTR = UTR 5' of the tranlsation initiation codon (ATG start codon)
    :   3'UTR = UTR 3' of the translation termination codon
    
*   variant
    :   a difference between the reference sequence and the sample sequence
