---
layout: default-md
title: Introduction
order: 1
---

# {{page.title}}

Discussions regarding the uniform and unequivocal description of sequence variants in DNA and protein sequences (mutations, polymorphisms) were initiated by two papers published in 1993; Beaudet AL & Tsui LC ([DOI paper](http://onlinelibrary.wiley.com/doi/10.1002/humu.1380020402/abstract) / [abstract](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=8401532&dopt=Abstract)) and Beutler E ([paper](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682427/pdf/ajhg00054-0240.pdf) / [abstract](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=8352285&dopt=Abstract)). The original suggestions presented were widely discussed, modified, extended and ultimately resulted in nomenclature recommendations that have been largely accepted and are applied world-wide ([_see History_](/history)).

Current rules (den Dunnen, JT and Antonarakis, SE (2000), [paper](http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART) / [abstract](http://www.ncbi.nlm.nih.gov/pubmed/10612815)) however do not extensively cover all types of variants and the more complex changes. These pages will list, based on the last publication, the existing nomenclature recommendations as well as the most recent suggestions (_in italics and marked [![](new.gif)](#del)_). More details regarding the  latest additions can be found at the [_Discussion page_](disc.html). These pages can be used as a guide to describe any sequence variant identified and should help to get a uniformly accepted standard.

Discussions regarding the advantages and disadvantages of the recommendations made are necessary in order to continuously improve the system. What is listed on these pages represents the current consensus of the discussions. We invite investigators to communicate to us regarding the recommendations as well as to send us complicated cases not yet covered, with a suggestion of how to describe these (E-mail to: [VarNomen @ HGVS.org](mailto:VarNomen%20@%20HGVS.org)).   

### Mutation and polymorphism

In some disciplines the term**"mutation"** is used to indicate "_a change_" while in other disciplines it is used to indicate "_a disease-causing change_". Similarly, the term **"polymorphism"** is used both to indicate "_a non disease-causing change_" or "_a change found at a frequency of 1% or higher in the population_". To prevent this confusion we do not use the terms mutation and polymorphism (including SNP or Single Nucleotide Polymorphism) but use neutral terms like **"sequence variant"**, **"alteration"** and **"allelic variant"**. The Vol.19(1) issue of Human Mutation (2002) contains several contributions discussing these issues as well as the fact that the term **"mutation"** has developed a negative connotation (see [Cotton RGH - p.2](http://onlinelibrary.wiley.com/doi/10.1002/humu.10029/pdf), [Condit CM et al. - p.69](http://onlinelibrary.wiley.com/doi/10.1002/humu.10023/pdf) and [Marshall JH - p.76](http://onlinelibrary.wiley.com/doi/10.1002/humu.10021/pdf)). Therefore, current guidelines of authorative organisations now also recommend to use the neutral term "**variant**" only (e.g. [Richards 2015, Genet.Med. 17:405-424](http://www.nature.com/gim/journal/v17/n5/pdf/gim201530a.pdf)).  

### ![](new.gif)Pathogenic

Another confusing term used frequently is "a **pathogenic** variant". While a non-expert concludes the variant described "_causes disease_", the expert probably means "_causes disease when in a specific context_"

*   causes disease when found in a male (X-linked recessive disorder)
*   causes disease when combined with a similar chance in the other allele (autosomal recessive)
*   causes disease when inherited from the father (imprinted)

To prevent confusion it therefore seems best not to use the term "_pathogenic_". A good alternative seems a neutral term like "**affects function**". In fact this properly describes what one actually means, the variant affects the normal function of the gene/protein (in whatever way). This also solves the issue of what term to use for non-disease phenotypes like skin/hair/eye colour or blood group. In such cases it is problematic to choose the phenotype to call "_norma_l" or "_pathogenic_". Using "**affects function**" is clear and effective. To classify variants people use most frequently 5 categories. Based on affects function these could be; _affects funtcion,  probably affects function,  unknown,  probably does not affect function (or probably no functional effect),  does not affect function (no functional effect)_. Variants for which a functional effect is unknown can together be called "**variants of unknown significance**" (**VUS**).