---
layout: recommendation
parent: DNA
title: alleles
definition: 
    Allele: a series of variants on one chromosome.
discussion:
    Was originally the recommendation to use the format [c.76A>C+c.83G>C]?: Indeed, originally <a href="http://dx.doi.org/10.1002/%28SICI%291098-1004%28200001%2915:1%3c7::AID-HUMU4%3e3.0.CO;2-N">den Dunnen and Antonarakis, 2000</a> the suggestion was to describe two changes in a gene on one chromosome as [c.76>C+c.83G>C], i.e. using a "+"-character to separate the two changes, while an earlier publication suggested to use a ";" ([c.76A>C;c.83G>C] <a href="http://dx.doi.org/10.1002/%28SICI%291098-1004%281998%2911:1%3c1::AID-HUMU1%3e3.0.CO;2-O">(Antonarakis and the Nomenclature Working Group, 1998</a>). To prevent confusion with older publications, to improve overall consistency and to keep descriptions as short as possible, the 2000 proposal was retracted. The recommended format is c.[76A>C;83G>C].
    In recessive diseases, is it important I show which variants were found in which combination?: When in one individual you find more then one variant it is essential that you clearly indicate which variant(s) were found and on which allele(s);<ul><li>disease severity will depend on the combination of variants found,</li><li>in recessive disease, when two variants are on one allele an individual is a carrier or you might not have found the variant on the 2nd allele.</li></ul>
    I find the notation c.[76A>C] without describing the second allele misleading; not enough researchers know this refers to only one of the two alleles present. Would using c.[76A>C];[] be OK?: No, the recommended description is LRG_199t1:c.[76A>C];[76=], i.e. c.76= for "no change" on the second allele. 
    How should I describe the variants detected in males and females for a gene on the X-chromosome?: In <b>females</b> the description is straightforward, like LRG_199t1:c.[76A>C];[76=]. In <b>males</b> there is no second allele (X-chromosome) which can be described as LRG_199t1:c.[76A>C];[0], i.e. using "<b>c.0</b>" to indicate the absence of a second X-chromosome.
    I have a patient with hearing loss and variants in the GJB2 (c.35delG) and GJB6 (c.689_690insT) genes, how should I describe this? <i>(Nancy Carson, Ottawa, Canada)</i>: The recommendation is to use the format GJB2:c.[35delG] GJB6:c.[689_690insT]. This uses standard HGVS descriptions and prevents confusion regarding which variant was found in which gene. Note it is essential that you also define the coding DNA reference sequence used. Another format, coping with this directly, is to describe the variants as NM_004004.2:c.[35delG] NM_006783.1:c.[689_690insT], i.e. using the Genbank reference sequences in stead of the HGNC approved Gene Symbol.
    When for a haplotype I have a range of changes to report, is there a suggested short format to use?: When it is once clearly specified (e.g. in the Materials & Methods) what variants are listed and based on which reference sequence(s), alleles may be reported in a simplified format like below. Variants should be listed in genomic order and using "[ ]" for variants on the same chromosome.<ul><li>haplotype with all variants in relation to several different reference sequences, both genomic and coding DNA<ul><li>description of the reference haplotype; [M59228.1:g.250G>C; AF209160.1:g.572CA[11_21]; Z11861.1:g.61T>C;Z16803.1:g.114A[18_22]]</li><li>short haplotype description; [C;13;T;21]</li></ul><li>haplotype with all variants in relation to one coding DNA reference sequence<ul><li>description of the reference haplotype; NM_004006.1:c.[837G>A; 1704+51T>C; 3734C>T;6438+2669T(16_23); 6571C>T; 7098+13212GT(15_19)]</li><li>short haplotype description; [G;C;C;18;T;17]</li></ul></ul>
    Suggestion (Peter Taschner, Leiden): Nomenclature recommendations mainly apply to genotype descriptions in tables. Unfortunately, these are not very useful in the general text of a paper. For instance, OPRM1:c.118A>G or rs1799971:A>G can be used to describe the variant, but in a paper you might like to discuss the phenotypic consequences of different genotypes. In fact the current recommendation is to use OPRM1:c.[118A>G];[118A=] to describe a heterozygote and OPRM1:c.[118A=];[118A=] and OPRM1:c.[118>G];[118>G] for the homozygotes. I would like to <b>suggest</b> to describe the genotypes in the text like;<ul><li>OPRM1:c.118AA homozygotes </li><li>OPRM1:c.118GA heterozygotes </li><li>OPRM1:c.118GG homozygotes </li> </ul>The different alleles could then be designated as the OPRM1:c.118A allele and the OPRM1:c.118G allele. In combination with variants of other genes, the genotype descriptions could be OPRM1:c.118AA, GJB2:c.76AC double heterozygotes, etc.
---
