---
layout: recommendation
parent: DNA
title: complex (HGVS/ISCN)
definition: 
    Complex: a sequence change where, compared to a reference sequence, a range of changes occur that can not be described as one of the basic variant types (substitution, deletion, duplication, insertion, conversion, inversion, deletion-insertion, or repeated sequence).
discussion:
    What is ISCN?: ISCN is short for the International System for human Cytogenetic Nomenclature, covering the description of numerical and structural chromosomal changes detected using microscopic and cytogenetic techniques. The recommendations are prepared by the International Standing Committee on Human Cytogenetic Nomenclature and published in collaboration with the journal <i><a href='http://www.karger.com/Journal/Home/224037'>Cytogenetic and Genome Research</a></i> (since 1963). The committee includes three members from the Americas, three from Europe, one from Asia, and one from Africa/Australia/Oceania. Members are elected for a 5 year period, unless developments demand earlier changes. The organization of the submission of nominations, ballots for voting and election of members is the responsibility of the existing Editor and Committee chair.<br>The latest recommendations, ISCN2013, were finalized by the ISCN2013 committee and its advisors at a meeting in Seattle (Washington) in April 2012. The ISCN2013 recommendations are available as <i><a href='http://www.karger.com/iscn2013'>book from Karger Publishers</a></i> (Eds. Lisa G Shaffer, Jean McGowan-Jordan & Michael Schmid). Questions/suggestions regarding the ISCN recommendations should be addressed to <b><i>Jean McGowan-Jordan</i></b> (Ottawa, Canada), chair of the ISCN committee.<br><ul><li><b>current ISCN committee</b><br><u>Chair:</u> Jean McGowan-Jordan (<i>Ottawa, Canada</i>)<br><u>Members:</u> Jaclyn Biegel (<i>Philadelphia, USA</i>), Myriam Chaabouni (<i>Tunis, Tunisia</i>), Johan T den Dunnen (<i>Leiden, Nederland</i>), Jin-Yeong Han (<i>Busan, South Korea</i>), Nils Mandahl (<i>Lund, Sweden</i>), Kathleen W Rao (<i>Chapel Hill, USA</i>), Annet Simons (<i>Nijmegen, Nederland</i>). <i></i><br><u>Advisors:</u> Cynthia C Morton (<i>Boston, USA</i>), Michael Schmid (<i>Wurzburg, Germany</i>)</li><li><b>ISCN2013 committee</b><br><u>Chair:</u> Lisa G Shaffer (<i>Spokane, USA</i>),<br><u>Members:</u> Jaclyn Biegel (<i>Philadelphia, USA</i>), Myriam Chaabouni (<i>Tunis, Tunisia</i>), Johan T den Dunnen (<i>Leiden, Nederland</i>), Jin-Yeong Han (<i>Busan, South Korea</i>), Nils Mandahl (<i>Lund, Sweden</i>), Jean McGowan-Jordan (<i>Ottawa, Canada</i>), Kathleen W Rao (<i>Chapel Hill, USA</i>), Annet Simons (<i>Nijmegen, Nederland</i>)<br><u>Advisors:</u> Lynda J Campbell (<i>Melbourne, Australia</i>), Michael Schmid (<i>Wurzburg, Germany</i>).</li></ul>
    Is the description NM_04006.1:c.123+45_123+51TSDinsL1.603bp acceptable (TSD = target site duplication, insL1 indicates the nature of the insert (L1, Alu or SVA), 603bp = the number of inserted base pairs)?: No, not realy, it is not exact. Following HGVS recommendations the description should be like NG_012232.1(NM_004006.1):c.123+45_123+51dupinsXXXXXX.x:g.393_1295. So give a genomic reference sequence to describe the intronic variant, use "dup" (not "TSD") and exactly describe the insertion, not like "insL1.603bp". In the example XXXXXX.x is a GenBank file (accession.version number) containing the inserted L1 sequence (nucleotides g.393_1295).
---
