---
parent: DNA
title: deletion
---

<p>A nucleotide deletion is a sequence change where one or more nucleotides
      are removed (<i>see <a href="standards.html#define">see Standards -
          Definition</a></i>). Deletions are described using <i><b>"del"</b></i>
      after an indication of the first and last nucleotide(s) deleted, separated
      by a "_" (<i>underscore</i>). For all descriptions the <em><b>most 3'
          position</b></em> possible is arbitrarily assigned to have been
      changed.<br>
      <i><b>NOTE: </b>to discriminate known variable sequences from other
        changes it is recommended to describe individual alleles differing from
        the reference sequence like g.210T[5] (preferred over g.210_211delTT) or
        g.121T[9] (preferred over g.210_211dupTT) </i>(<i><a href="#var">see
          Repeated sequences</a></i>).&nbsp; </p>
    <ul>
      <li>c.76_78del (alternatively c.76_78delACT) denotes a ACT deletion from
        nucleotides 76 to 78</li>
      <li>deletions with <em><strong>uncharacterised breakpoints</strong></em>
        (<i><a href="uncertain.html">see Uncertainties</a></i>)
        <ul>
          <li>c.(87+1_88-1)_(923+1_924-1)del denotes a deletion of exons 3 to7
            starting at an unknown position in intron 2 (between coding DNA
            nucleotides 87+1 and 88-1) and ending at an unknown position in
            intron 7 (between coding DNA nucleotides 923+1 and 924-1). The
            description indicates that exons 2 and 8 have been tested and shown
            not to be deleted<br>
            <span style="font-weight: bold; font-style: italic;">NOTE:</span>
            the description c.88-?_923+?del does not specify start/end of the
            deletion and is not correct when flanking sequences have been tested
            <em><strong></strong></em> (<i><a href="http://www.hgvs.org/mutnomen/uncertain.html#exondel">see
                Uncertainties</a></i>) </li>
          <li>c.(?_-30)_(*220_?)del denotes the deletion of the entire gene
            (coding DNA reference sequence running from -30 (cap site) to *220
            (polyA-addition site)</li>
          <li><img src="new.gif" border="0" height="12" width="31">c.88+101_oGJB2:c.355-1045del
            denotes a deletion which ends in the flanking GJB2 gene at position
            355-1045 (in the intron between nucleotides 354 and 355) on the
            reverse strand (the genes are thus located and fused in opposite
            transcriptional directions,<em> </em><i><a href="disc.html#fusion">see
                Discussion</a></i>)</li>
        </ul>
      </li>
      <li><a name="except"></a>for all descriptions the <b>most 3' position</b>
        possible is arbitrarily assigned to have been changed (<a href="FAQ.html#most3"><i>see
            FAQ</i></a>);
        <ul>
          <li>ACTT<b>TGT</b>GCC to&nbsp;ACTTGCC is described as c.5_7del
            (c.5_7delTGT, not as c.4_6delTTG)</li>
          <li>ctt<b>t</b>agGCATG to cttagGCATG in an intron is described as
            c.301-3delT (not as c.301-5delT)</li>
          <li>TCACTG<b>TCTG</b>CGGTAATC to TCACTG CGGTAATC is described as
            c.7_10del (c.7_10delTCTG) and not as c.4_7del (c.4_7delCTGT).</li>
          <li>AAAG<b>AAGAG</b>GAG to AAAG GAG is described as c.5_9del
            (c.5_9delAAGAG) and not as c.3_7del (c.3_7delAGAAG)</li>
          <li><b><i>Exceptions</i></b>
            <ul>
              <li>using a <i>coding DNA reference sequence</i> there is <b><i>an
                    exception</i></b> to the rule around exon/intron and
                exon/exon borders when identical nucleotides flank the
                exon/intron or exon/exon border;
                <ul>
                  <li>when the exon 3/intron 3 border is ..CAGgtg.. and RNA
                    analysis shows no effect on splicing but a deletion of a G
                    the change ..CAGgtg.. to ..CAgtg.. is described as c.3delG <i><b>and
                        not</b></i> c.3+1delG.</li>
                  <li>when exon 3 ends with ..CAA.. and exon 4 starts with
                    ..ACG.. and the sequence of genomic DNA shows that the last
                    A-nucleotide of exon 3 is deleted (and not the first
                    A-nucleotide in exon 4), the deletion changing ..CAAACG.. to
                    ..CAACG.. is described as c.3delA <b><i>and not</i></b>
                    c.4delA</li>
                </ul>
              </li>
              <li>c.1210-12T(5_9) (<i>not c.1210-6T(5_9)</i>) describes the
                variable stretch of 5 to 9 T-residues in intron 9 of the CFTR
                gene. The most commonly used CFTR coding DNA reference sequence
                contains a stretch of 7 T's (<i><a href="#var">see Repeated
                    sequences</a></i>).&nbsp;</li>
            </ul>
          </li>
        </ul>
      </li>
    </ul>