---
layout: recommendation
parent: DNA
title: deletion
definition: 
    Deletion: a sequence change where, compared to a reference sequence, one or more nucleotides are not present (deleted).
discussion:
    Can I use g.123del6 to describe a 6 nucleotide deletion?: No, a deletion of more than one residue should mention the first and last residue deleted, separated using the range symbol ("_", underscore), e.g. g.123_128del and not g.123del6.<a name="6del"></a>
    In the example above, <b>c.3delA</b>, should the description based on a coding DNA reference sequence not be c.4delA?: Strictly speaking you are right. However, for cases like this an exception was made to prevent that when c.4delA is translated back to a genomic position one would end up at the wrong nucleotide, even in the wrong exon (<i>see exception in <a href='http://varnomen.HGVS.org/bg-material/numbering/#DNAc'>Numbering</a></i>).
    Is the description of a deletion of exon 17 as c.EX17del still allowed?: A description like c.EX17del has never been allowed. Descriptions should be specific and indicate the nucleotides affected by the change.
    Deletions in the BRCA1 gene are usually mediated by Alu sequences having a very high homology, reaching 100% in the breakpoint region. In such cases, what nucleotide should be used to describe the deletion breakpoint?: In cases like this the 3'rule applies (<a href='http://www.HGVS.org/varnomen/recommendations/general/'><i>see Recommendations General</i></a>), i.e. the deletion breakpoint is determined by the first nucleotide that differs after shifting the alignment as far 3' as possible. The first nucleotide differing is the <b>first nucleotide deleted</b>.
    PCR analysis of a gene on the X-chromosome shows products for exons 1_3, no product is detected for exons 4_14 (exon 14 is the last exon of the gene). Since PCR fails already when one primer is not hybridising, we are not sure whether exon 4 and 14 are completely absent, or only partially. To describe the deletion I would therefore like to use the last base of exon 3 with "+?" and the last base of exon 13 with a "+?. What are your recommendations? (<i>Erik-Jan Kamsteeg, Nijmegen, Nederland</i>): Literally speaking you are right and it is best to set the borders as precise as possible. When exon 3 is present the location of the reverse primer can be used to set the most 5' border (something like c.987+123). However, for the 3' end your reasoning does not make a difference. Since you do not know how far the deletion extends, you have no positive PCR limiting the deletion at the 3' end, using the location of exon 13 since exon 14 <b>might be present</b> would give the wrong impression. Consequently the precise description can only be like c.(987+123_?)del. Is this realy more informative then c.(987+1_?)del, using the exon 3 exon/intron border?
    In literature I often see the description "deltaF508" for a variant in the CFTR gene in patients with Cystic Fibrosis. Is the variant detected in these patients c.1522_1524delTTT?: No. The sequence surrounding amino acid Phe508 in the CFTR gene is ..-ATC-TTT-GGT-.. (c.1519 to c.1527). Three different deletions (TC-T, C-TT and -TTT-) would give the reported protein variant "Phe508del". Applying the 3' rule [_see Recommendations_](/recommendations/general/) yields two different changes at DNA level, c.1521_1523del and c.1522_1524del. When you assume the change at DNA level is c.1522_1524delTTT, deletion of exactly the Phe508 encoding triplet, you are wrong. The change found in patients is mostly c.1521_1523delCTT. So, without a proper description in the manuscript one can not be certain.
    Suggest to use "los" for a loss from a mononucleotide stretch: <u>Pat O'Neill (Burlington, USA) writes</u>; I especially like the use of "dup" in place of "ins" when the insertion creates a run of two or more nucleotides. I feel that there should be a parallel term for the loss of a nucleotide from a run of two or more instead of just "del". This is because of the mechanistic implications of both an ins and a del of a nucleotide in a run. Has this been discussed? My thought for a term in place of "del" is <b>"los"</b>for loss.<br><u>Shuji Ogino (Boston, USA)</u> agrees but suggests to use <b>"dec"</b> for a decrease in length.<br><u>Reply (JdD)</u>; The "dup" nomenclature was introduced because it is <b>simpler, shorter and less unequivocal</b> (see above). The potential mechanistic relation is nice but was not decisive. Basically a description should be clear/unequivocal and it is not intended to contain other information.
---
